apps/tallycoin-connect/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "3.7"
+
+services:
+  web:
+    image: dennisreimann/tallycoin_connect:v1.7.0
+    restart: on-failure
+    stop_grace_period: 1m
+    ports:
+      - "$APP_TALLYCOIN_CONNECT_PORT:$APP_TALLYCOIN_CONNECT_PORT"
+    volumes:
+      - ${LND_DATA_DIR}:/lnd:ro
+      - ${APP_DATA_DIR}/data:/data
+    environment:
+      LND_SOCKET: "$LND_IP:$LND_GRPC_PORT"
+      LND_MACAROON_PATH: "/lnd/data/chain/bitcoin/$BITCOIN_NETWORK/admin.macaroon"
```

Comment:
Do we need to pass in the admin macaroon here or would the invoice macaroon be sufficient?

Reply:
I believe the dependency ln-service requires the admin macaroon, although Tallycoin Connect does not require (or use) full permissions.

Reply:
Ok got it!

This doesn't need to be a blocker for the initial release, it's fine to as is, but it would be great to see a future version of Tallycoin Connect that works with just an invoice macaroon. 😍